pyproject.toml

```diff
@@ -10,6 +10,11 @@ skip_gitignore = true
 line-length = 120
 target-version = ["py39"]
 
+# https://bandit.readthedocs.io/en/latest/config.html
+[tool.bandit]
+tests = []
+skips = ["B101"]
```

Comment:
Can you please document why `"B101"` is skipped?

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/77/commits/d76ac3f322276ec68974c289985ce48e435c7253.